scripts/seastar-addr2line

```diff
@@ -99,13 +99,14 @@ class BacktraceResolver(object):
                 addresses = []
                 for obj in m.group(2).split():
                     m = re.match(self.address_re, obj)
+                    #print(f"  >>> '{obj}': address {m.groups()}")
                     addresses.append({'path': m.group(1), 'addr': m.group(2)})
                 ret['addresses'] = addresses
                 return ret
 
             m = re.match(self.asan_re, line)
             if m:
-                #print(f">>> '{line}': asan {m}")
+                #print(f">>> '{line}': asan {m.groups()}")
```

Comment:
A better alternative to commenting out is to have a flag which controls execution. Not a blocker.